srcpkgs/cargo-auditable/template

```diff
@@ -4,6 +4,9 @@ version=0.5.2
 revision=1
 build_wrksrc=cargo-auditable
 build_style=cargo
+# Needed so that we don't try to build cargo-auditable with cargo-auditable.
+# There's ways around this, but I'm not sure which one to take yet
+make_cmd=cargo
```

Comment:
We can either build cargo-auditable twice in the package and do magic PATH fuckery, or we could have a cargo-auditable-bootstrap package which mirrors the regular cargo-auditable package, with the only addition of having a `make_cmd=cargo`. Either feels a bit icky.